CONTRIBUTING.md

```diff
@@ -17,6 +17,8 @@ under the terms of [COPYING](COPYING), which is an MIT-like license.
   (pkg-name | nixos/<module>): (from -> to | init at version | refactor | etc)
 
   (Motivation for change. Additional information.)
+
+  (breaks things: <A brief description of what breakage is introduced.>)
```

Comment:
i'm suggesting `breaks things:` because that string currently does not occur in any nixpkgs commit (as reported by `git log --grep "breaks things:"`)

Reply:
hmm, @joepie91 mentioned https://www.conventionalcommits.org/en/v1.0.0/
1 commit with a `BREAKING CHANGE:` footer already exists, and there are no other uses of that string